### ISSUE TITLE
error in file jsb_opengl.js  

### ISSUE THREAD

**Issue by zAncel** [cocos2d/cocos2d-x#16181]
bellow three funcion, the parameter name is **bufer** ,but in function ,actually use is **buffer** 

``` javascript
gl.deleteBuffer = function(bufer) {
    var buffer_id = buffer.buffer_id;
    // Accept numbers too. eg: gl.deleteBuffer(0)
    if( typeof buffer === 'number' )
        buffer_id = buffer;

    gl._deleteBuffer(buffer_id);
};

gl.deleteRenderbuffer = function(bufer) {
    var buffer_id = buffer.renderbuffer_id;
    // Accept numbers too. eg: gl.deleteRenderbuffer(0)
    if( typeof buffer === 'number' )
        buffer_id = buffer;

    gl._deleteRenderbuffer(renderbuffer_id);
};

gl.deleteFramebuffer = function(bufer) {
    var buffer_id = buffer.framebuffer_id;
    // Accept numbers too. eg: gl.deleteFramebuffer(0)
    if( typeof buffer === 'number' )
        buffer_id = buffer;

    gl._deleteFramebuffer(buffer_id);
};
```

**Comment by minggo** [comment]
Yep, i think it is a bug. Can it work correct after changing from `bufer` to `buffer`?

**Comment by zAncel** [comment]
yes, it is work

2016-07-25 10:54 GMT+08:00 minggo notifications@github.com:

**Comment by minggo** [comment]
@zAncel why not send a pull request?

**Comment by minggo** [comment]
fixed in #16269
